package.json

```diff
@@ -24,18 +24,18 @@
   },
   "dependencies": {
     "ansi": "^0.3.1",
-    "bplist-parser": "^0.1.0",
+    "bplist-parser": "^0.2.0",
     "cross-spawn": "^6.0.5",
-    "elementtree": "0.1.7",
-    "endent": "^1.1.1",
-    "fs-extra": "^8.0.0",
-    "glob": "^7.1.2",
-    "minimatch": "^3.0.0",
+    "elementtree": "^0.1.7",
+    "endent": "^1.3.0",
+    "fs-extra": "^8.1.0",
+    "glob": "^7.1.4",
+    "minimatch": "^3.0.4",
     "plist": "^3.0.1",
     "q": "^1.4.1",
     "strip-bom": "^4.0.0",
-    "underscore": "^1.8.3",
-    "which": "^1.3.0"
+    "underscore": "^1.9.1",
+    "which": "^1.3.1"
```

Comment:
There already is `which@2`, but if we remove it in #102 this will not matter anymore

Reply:
👍